extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc

```diff
@@ -1118,6 +1123,12 @@ Using these embedded relationships ensures that an incomplete sequence cannot be
 |[stixtype]#{location_url}[location]#
 |The task occurred at a specific location.
 
+
+|[stixtype]#<<task,task>>#
+|[stixrelationship]#performed-by#
+|[stixtype]#{identity_url}[identity]#
+|The task was performed by the identity.
+
```

Comment:
there is an identity "performed" task relationship in Reverse Relationships section.

Reply:
dropped it